README.md

```diff
@@ -110,86 +110,122 @@ bash test/test_cleanup.sh
-tested, you can go ahead and run the workflows on your samples.
+Now that your virtual environment is set up and the workflow is deployed and
+tested, you can go ahead and run the workflow on your samples.
 
-But first, here is a brief description of what each of the three workflows
-does:
+But first, here is a brief description of what the workflow does:
 
 ### Workflow description
 
-The repository contains the following workflows, all implemented in Snakemake
-and fully containerized:
+The workflow consist on one main `Snakefile` 
+gathering three different subworkflows,
+**_PREPARE_**, **_MAP_** and **_QUANTIFY_**. 
+Each one of them is implemented in an individual Snakemake file
+and fully containerized.    
+
 
 #### _PREPARE_
 
-The first workflow, **_PREPARE_** downloads and processes "genome resources"
+The first subworkflow, **_PREPARE_**, downloads and processes "genome resources"
 from the publicly available repositories [Ensembl][ensembl] and
 [miRBase][mirbase] according to your instructions. Resources are then processed
 to prepare indexes and other contingent resources that will be used in later
 steps.
 
-The scheme below is a visual representation of an example run of the
-**_PREPARE_** workflow:
-
-> ![rule-graph-prepare][rule-graph-prepare]
-
 #### _MAP_
 
-The second workflow, **_MAP_** aligns the user-provided short read smallRNA-seq
-libraries against the references generated with the **_PREPARE_** workflow. For
+The second subworkflow, **_MAP_**, aligns the user-provided short read smallRNA-seq
+libraries against the references generated with the **_PREPARE_** subworkflow. For
 increased fidelity it uses two separate aligning tools, [Segemehl][segemehl]
 and our in-house tool [Oligomap][oligomap]. In both cases, reads are aligned
 separately to the genome and the transcriptome. Afterwards, alignments are
 merged in a way that only the best alignment (or alignments) of each read are
 kept.
 
-The scheme below is a visual representation of an example run of the **_MAP_**
-workflow:
-
-> ![rule-graph-map][rule-graph-map]
-
 #### _QUANTIFY_
 
-The third and final workflow, **_QUANTIFY_** quantifies miRNA expression by
-intersecting the alignments from the **_MAP_** workflow with the annotations
-generated in the **_PREPARE_** workflow. Intersections are computed with
+The third and final subworkflow, **_QUANTIFY_**, quantifies miRNA expression by
+intersecting the alignments from the **_MAP_** subworkflow with the annotations
+generated in the **_PREPARE_** subworkflow. Intersections are computed with
 [`bedtools`][bedtools] for one or multiple of mature, primary transcripts and
 isomiRs. Reads consistent with each miRNA are counted and tabulated.
 
-The scheme below is a visual representation of an example run of the
-**_QUANTIFY_** workflow:
 
-> ![rule-graph-quantify][rule-graph-quantify]
+The schema below is a visual representation of
+an example run of the whole workflow:
```

Comment:
I know you didn't write this, but this is not correct. It _would_ be if you showed the workflow DAG instead of the rule graph. The rule graph is a representation of how the different rules are linked in a Snakemake workflow. It is a simplification of the DAG of an actual run (check the Snakemake docs to understand the difference).